operator_tracker/src/main.rs

```diff
@@ -0,0 +1,154 @@
+use std::sync::Arc;
+
+use axum::body::Body;
+use axum::extract::State;
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::routing::post;
+use axum::Json;
+use axum::Router;
+use ethers::providers::{Http, Provider, ProviderExt};
+use ethers::types::Address;
+use log::debug;
+use log::info;
+use operator_tracker::create_or_update_operator_version;
+use operator_tracker::OperatorVersion;
+use operator_tracker::OperatorVersionPayload;
+use operator_tracker::RegistryCoordinator;
+use operator_tracker::RegistryCoordinatorContract;
+use sqlx::postgres::{PgPool, PgPoolOptions};
+use sqlx::Executor;
+
+// The RegistryCoordinator contract
+// This is the contract that the service will interact with to get the status of the operator
+// The response is a uint8 that represents the status of the operator
+// 0 - Never registered
+// 1 - Registered
+// 2 - Deregistered
```

Comment:
I don't really know how ethers handles this but couldn't we just use the contract enum?